provider/secrets_manager_provider.go

```diff
@@ -31,10 +29,15 @@ type SecretsManagerProvider struct {
 	clients []SecretsManagerClient
 }
 
+type SecretsManagerGetDescriber interface {
```

Comment:
Same story on `secretsmanageriface`, just a custom interface to satisfy the type for easier testing